test/src/integration/integration_test.go

```diff
@@ -189,3 +192,17 @@ func gitClone(t *testing.T, repo string, moduleDir string) {
 
 	shell.RunCommand(t, gitCommand)
 }
+
+func runHelmDelete(t *testing.T) {
+	bastionIP := lookupTargetValue(t, "network", "bastion_ip")
+
+	publicHost := ssh.Host{
+		Hostname:    bastionIP,
+		SshAgent:    true,
+		SshUserName: "centos",
+	}
+
+	commandHelmDelete := "helm delete prod"
```

Comment:
Other variables in this file are named `*Command`, so this might be better named `helmDeleteCommand`.

Reply:
👍 fixed in d0060b9